src/movies.js

```diff
@@ -1,16 +1,76 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+
+
+
+const testArr = [
+    {
+      title: 'Paths of Glory',
+      year: 1957,
+      director: 'Stanley Kubrick',
+      duration: '1h 28min',
+      genre: ['Drama', 'War'],
+      rate: 8.4
+    },
+    {
+      title: 'Django Unchained',
+      year: 2012,
+      director: 'Quentin Tarantino',
+      duration: '2h 45min',
+      genre: ['Drama', 'Western'],
+      rate: 8.4
+    }
+ ]
+
+function getAllDirectors (moviesArr) {
+  const directors = moviesArr.map(function(movie){
+   return movie.director
+  })
+  return directors
+}
+
+getAllDirectors(testArr);
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-
+function howManyMovies(movies) {
+  const filterd = movies.filter(function(movie)  {
+      return movie.director === 'Steven Spielberg' &&
+      movie.genre.includes('Drama')
+  })
+}
```

Comment:
this function needs a `return filterd` to get a response. At the moment you're not getting anything back